contracts/multisig/src/events.rs

```diff
@@ -1,31 +1,32 @@
 use std::collections::HashMap;
 
-use cosmwasm_std::{Addr, HexBinary, Uint64};
+use axelar_wasm_std::IntoEvent;
+use cosmwasm_std::Addr;
 use router_api::ChainName;
-use serde_json::to_string;
 
 use crate::key::{PublicKey, Signature};
 use crate::types::MsgToSign;
 
+#[derive(IntoEvent)]
 pub enum Event {
     // Emitted when a new signing session is open
     SigningStarted {
-        session_id: Uint64,
+        session_id: u64,
```

Comment:
@cjcobb23 is it fine I changed this to `u64`? That means it would be serialized as a number rather than a string.